reactioncommerce/reaction
# Fix issue where router would initialize before primaryShopId was available

Paired with @spencern on this, and after some struggle, a simple solution appears before us.

- Add a check to wait on reactive var `Reaction.primaryShopId` in addition to the subscriptions before initializing the router.

To test:
- Sign in/out ~10 times
- Hard refresh ~10 times

While doing the above, observe no grey screen, no missing components and the app still functions as expected.
